### ISSUE TITLE
Unable to update share object from mobile app

### ISSUE THREAD

**Issue by dhsc19** [ampache/ampache#802]
I can create and delete a share object in DSub and Ultrasonic, but when I try to "update information" I get "Failed to update playlist information for share.php.  The requested data was not found."  My apache log shows a 200 OK, but I'm not seeing any activity in ampache's log.

xxx.xxx.xxx.xxx - - [09/Apr/2015:15:12:42 -0700] "POST /ampache/rest/updateShare.view?u=carlo&p=enc:536e61466c6f73703139&v=1.2.0&c=DSub&id=42&description=test&expires=0 HTTP/1.1" 200 28156 "-" "-"

**Comment by dhsc19** [comment]
Ultrasonic is giving me an pop-up error message in the share screen:

Invalid long: ""

**Comment by Afterster** [comment]
I tried with DSub successfully and I don't have any error on Ultrasonic.
Could you copy/past your Ampache/Apache logs and xml output? Thanks.

**Comment by dhsc19** [comment]
Hmm...ok.  For some reason, shares that I initially created on the web interface didn't show up in DSub until I created one on DSub first.  Afterwards, they're all showing up now.  Working good now.  On a side note, Ultrasonic appears to be gone from Google Play Store.  I tried uninstalling it to re-install and when I went back to look for it, it was gone.  Even third party links pointing to it end up on invalid URL now.  It was a good one to double check with, but I think once the next DSub update goes though, all the kinks will be ironed out.
